### ISSUE TITLE
Whats the meaning of "Batterie wird mehr als %" ?

### ISSUE THREAD

**Issue by JanSchaefer79** [biemond/solaredge.modbus#91]
Hello, i have a Sungrow Hybrid with Battery.
What is the meaning of function 
If  "Batterie wird mehr als %" 
If "Batterie wird weniger als %" ?

Is it the stored kwh in percent or is it the Battery Quality in percent ?

Is there somewhere discription for the German Cards ?

with best Regards

**Comment by biemond** [comment]
Hey , 

should be SOC flow trigger and not SOH. I think these are auto generated by homey , not by me and I guess it uses battery %

**Comment by JanSchaefer79** [comment]
Ok, i will try later again. Thanks for your comment. The translation is sometimes little bit confusing with the cards

**Comment by biemond** [comment]
yes , these are auto generated but most of my flow triggers are translated by your countryman.
